vendor/github.com/google/btree/btree.go

```diff
@@ -796,7 +796,7 @@ func (t *BTree) DescendLessOrEqual(pivot Item, iterator ItemIterator) {
 }
 
 // DescendGreaterThan calls the iterator for every value in the tree within
-// the range [last, pivot), until iterator returns false.
+// the range (pivot, last], until iterator returns false.
```

Comment:
Since we're doing a go mod replace of prometheus, only prometheus changes in `vendor/` should appear.